packages/cli/src/zosfiles/create/ds/ds.handler.ts

```diff
@@ -20,11 +20,23 @@ import { generateZosmfOptions } from "../Create.utils";
 
 export default class DataSetHandler extends ZosFilesBaseHandler {
     public async processWithSession(commandParameters: IHandlerParameters, session: AbstractSession): Promise<IZosFilesResponse> {
-        return Create.dataSetLike(
-            session,
-            commandParameters.arguments.dataSetName,
-            commandParameters.arguments.like,
-            generateZosmfOptions(commandParameters.arguments)
-        );
+        if (commandParameters.arguments.like == null) {
+            return Create.dataSet(
+                session,
+                CreateDataSetTypeEnum.DATA_SET_BLANK,
```

Comment:
Note: this is not a request for changes. 
The code is working as expected and letting z/osmf decide what it needs to do with the information.

I still want to post the following question:
- Should this dsntype be determined based on the provided arguments ?

Based on the help, it feels that someone should be able to specify PDS and a `DATA_SET_PARTITIONED` enum should be selected (and its defaults)

---

The only way I could find out this worked is by creating datasets with all allowed values, and here are my results:
- BASIC: `dsorg: PS`
- EXTPREF: `dsorg: PS-E`
- EXTREQ: `dsorg: PS-E`
- HFS: `dsorg: PS`
- LARGE: `dsorg: PS-L`
- PDS: `dsorg: PS` (I expected this to be `dsorg: PO` - might be a z/osmf issue)
- LIBRARY: `dsorg: PS` (I expected this to be `dsorg: PO` - might be a z/osmf issue)
- PIPE: `dynamic allocation error` 🤷 

---
Help text:
```
   --data-set-type | --dst | --dsntype (string)

      The data set type (BASIC, EXTPREF, EXTREQ, HFS, LARGE, PDS, LIBRARY, PIPE)
```